README.md

```diff
@@ -43,20 +43,27 @@ Configure plugin:
       <configuration>
         <chartDirectory>${project.basedir}</chartDirectory>
         <chartVersion>${project.version}</chartVersion>
-        <helmRepoUrl>${helm.repo.url}</helmRepoUrl>
-        <helmUploadUrlStable>${helm.repo.url}/stable/api/charts</helmUploadUrlStable>
-        <helmUploadUrlSnapshot>${helm.repo.url}/snapshot/api/charts</helmUploadUrlSnapshot>
+        <uploadRepoStable>
+            <name>stable-repo</name>
```

Comment:
Why did you decide to use a baseUrl with fixed paths for stable and snapshot, instead of configuring both addresses?

Reply:
I see it. Missread the diff...